cli/src/main/kotlin/commands/AdvisorCommand.kt

```diff
@@ -76,10 +78,10 @@ class AdvisorCommand : CliktCommand(name = "advise", help = "Check dependencies
 
     private val advisorFactory by option(
         "--advisor", "-a",
-        help = "The advisor to use, one of ${Advisor.ALL}"
+        help = "The advisor to use, one of ${allAdvisorsByName.keys}"
```

Comment:
This code triggered an alignment PR on my side, see https://github.com/oss-review-toolkit/ort/pull/3765. Could you please align to that style?